src/lib/components/Calendars/util.ts

```diff
@@ -0,0 +1,7 @@
+export type EventLike = {
```

Comment:
if these are the raw types provided by Event, grab them from prisma instead.

Reply:
I thought we had an issue with Prisma and ESM, which is why we weren't pulling things directly from Prisma.